x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms.tsx

```diff
@@ -183,26 +185,33 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn, 'field
           fullWidth
         >
           <FixedEuiRange
-            min={1}
-            max={20}
+            min={MIN_NUMBER_OF_VALUES}
+            max={MAX_NUMBER_OF_VALUES}
             step={1}
             value={currentColumn.params.size}
             showInput
             showLabels
             compressed
             onChange={(
               e: React.ChangeEvent<HTMLInputElement> | React.MouseEvent<HTMLButtonElement>
-            ) =>
+            ) => {
+              let value = Number((e.target as HTMLInputElement).value);
+              if (value > MAX_NUMBER_OF_VALUES) {
```

Comment:
Nitpick: Can we make these shorter?
```suggestion
value = Math.min(MAX_NUMBER_OF_VALUES, Math.max(value, MIN_NUMBER_OF_VALUES));
```

Reply:
Don't know if it is possible to replace multiple lines with a suggestion :)

Reply:
@dej611 thanks for your feedback, I completely changed this PR based on your feedback – please re-check when you find a moment. Now it's possible to input incorrect values (empty strings or numeric), but they won't update the visualization/won't send request to backend.